### ISSUE TITLE
Document 3rd party connections

### ISSUE THREAD

**Issue by coreyward** [SoftFever/OrcaSlicer#6442]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

Orca Slicer reaches out to various IP addresses and websites over the network on ports 80, 8000, 8080, and 10001 (possibly others). There are a handful of clear requests to Bambu or AWS ELB servers, but the nature of most of the remaining requests is unclear. Many do not have any DPI-indicated hostnames associated, and both the data being transferred as well as who it is being sent to are unclear. For example, I have observed requests to connect to `chats.partialqyrlkfa.com`, `64.42.179.82`, `173.234.30.210`, `118.107.244.104`, and many others.

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

Is it possible to get these connections documented, including the nature of the information being sent and the purpose it is being sent for? It would also be helpful to know which are required for the software to operate (or which functions depend on each connection).

### Describe alternatives you've considered

Using other slicers that don't send data to unidentified third parties unannounced.

**Comment by gleep52** [comment]
I am currently attempting to figure out why my print jobs seem to send to BambuLabs' servers fine, but then take minutes to "download" them to the printer.  Looking at my firewall I am seeing a lot of this same strange traffic associated with my printer.  I have not yet tested this same issue with the official Bambu Studio app - but am curious if it's the servers at BambuLabs that use this strange domain chats.partialqyrlkfa.com and the other misc IPs?

**Comment by coreyward** [comment]
Sure is stale because no maintainers have chimed in. 👀

**Comment by coreyward** [comment]
Not stale, just ignored. Almost like @SoftFever has something to hide…

**Comment by nwalke** [comment]
Sad to see this doesn't have any response.  I want to move my printers to a separate VLAN and need to know what ports to allow from my user VLAN to the printers.

**Comment by itopspl** [comment]
This is still happening, I use a printer in my company and am getting detections from Microsoft Endpoint Security now.
I'd hate to have to go back to using Bambu Studio but the admins may force me.

@SoftFever could you chime in on this?

**Comment by seanchristians** [comment]
@itopspl sorry about this, but Bambu Studio is no better. I'm here because Defender flagged it for reaching out to one of the IPs in the issue description.

It seems like it's been about a year since this issue was created. I guess I'll just block these IPs.

**Comment by SoftFever** [comment]
Can you provide more information?  
Orca will only ping GitHub to check for updates.  

If you use Bambu, their closed-source plugin will connect to their servers.  
However, I do not believe it will connect to chats.partialqyrlkfa.com such address or similar websites. (Although I can't say it 100% sure as it's closed source) 

Could you please double-check if you have configured any of your printer's host names or Device UI to such an address?
<img width="887" height="575" alt="Image" src="https://github.com/user-attachments/assets/02e0dc95-a472-43fd-98b3-3692e76d8ad1" />

**Comment by pieterdevriesch** [comment]
I agree that it's the Bambu plugin, Bambu Studio gives exactly the same activity, and in both it only happens when you try to start a video stream. Which still works despite the blocked connections funnily enough. 

So I guess this issue can be closed since it's Bambu's issue, there's an issue on their git about this already now. Thanks for the update.

Edit: oops forgot this is my other account, the one you replied to is my work account.

**Comment by SoftFever** [comment]
Yeah, it's Bambu's plugin
[#7699](https://github.com/bambulab/BambuStudio/issues/7699)
[#7781](https://github.com/bambulab/BambuStudio/issues/7781)

If you're concerned, you can enable LAN mode on Bambu machines.
